test/e2e/windows/density.go

```diff
@@ -0,0 +1,306 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package windows
+
+import (
+	"fmt"
+	"sort"
+	"sync"
+	"time"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/kubernetes/test/e2e/framework"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+	"k8s.io/apimachinery/pkg/util/uuid"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Why are we using dot imports?

It seems deeply unlikely that ginkgo or gomega imports the windows package, so this is not an appropriate use of this functionality:

https://github.com/golang/go/wiki/CodeReviewComments#import-dot

Reply:
According to ginkgo's docs, https://onsi.github.io/ginkgo/#avoiding-dot-imports, its _rare to avoid the dot imports... ginkgo bootstrap and generate default to dot imports. I can omit the dot, but seems better to not have to preface everything with ginkgo/gomega